### PR TITLE
count successes on each conftest run

### PR DESCRIPTION
**Pull request by joejstuart**

### CODE REVIEW

**Review by zregvart — APPROVED** [review]
LGTM, but the unit tests need to be updated, most likely the acceptance tests as well...

File: internal/applicationsnapshot/report.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ type Component struct {
 	Violations []conftestOutput.Result  `json:"violations"`
 	Warnings   []conftestOutput.Result  `json:"warnings"`
 	Success    bool                     `json:"success"`
+	Passed     int                      `json:"passed"`
```
Comment (joejstuart) [inline internal/applicationsnapshot/report.go:38]: `Successes` or `Passed`? I'm leaning towards `Passed`

Comment (lcarva) [reply internal/applicationsnapshot/report.go:38]: Maybe `successes` because that's also a noun like `warnings` and `violations`?

Comment (joejstuart) [reply internal/applicationsnapshot/report.go:38]: This is a good point. Conftest uses Passed, but I can't find a good reason.

Comment (simonbaird) [reply internal/applicationsnapshot/report.go:38]: One note: We want to (in the future) actually list details about the successes. So maybe reserve `successes` for that purpose and use something like SuccessCount for this.

Comment (simonbaird) [reply internal/applicationsnapshot/report.go:38]: `Passed` isn't so bad though, idk.

Comment (lcarva) [reply internal/applicationsnapshot/report.go:38]: Oh! I thought this was about adding the list of passing rules, not just the count. +1 to Simon's suggestion.

**Review by simonbaird — APPROVED** [review]
Lgtm, assuming you can get the tests back to passing.

**Pull request merged**